### PR TITLE
fix product-page footer

### LINKED ISSUE #11130: Problems in the admin side on the bottom toolbar of product pages under 1039px
Hi
if you go in a product page in the back office
try to resize the borwser's width under 1039px you will see the the bottom toolbar will not be full size
it should have ad width of 100% because the lateral menu is not visible

**Screenshots**

**Additionnal information**
PrestaShop version: 1.7.4.3
PHP version: 7

bye

### PR DESCRIPTION
**Pull request by dineshbadrukhiya**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Problems in the Back-Office on the bottom toolbar of product page.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? |  Fixes #11130
| How to test?  |

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12904)

### CODE REVIEW

**Comment by dineshbadrukhiya** [comment]
@Quetzacoalt91 I have tried to compile assets but found so many files has been changed. It seems like the issue #11514

**Comment by matks** [comment]
> @Quetzacoalt91 I have tried to compile assets but found so many files has been changed. It seems like the issue #11514

Are you compiling on windows ? If yes, this is a known issue of our current webpack configuration 😭

**Comment by dineshbadrukhiya** [comment]
> > @Quetzacoalt91 I have tried to compile assets but found so many files has been changed. It seems like the issue #11514
> 
> Are you compiling on windows ? If yes, this is a known issue of our current webpack configuration

Yes, I'm compiling on windows.

**Comment by matks** [comment]
@dineshbadrukhiya We have found out that Webpack mistakes the OS for a file change and consequently recompile everything. Can I fix this PR for you by recompiling the assets ?

And if you feel like diving into webpack configuration to fix this behavior, you're welcome ! 😄

**Comment by dineshbadrukhiya** [comment]
> @dineshbadrukhiya We have found out that Webpack mistakes the OS for a file change and consequently recompile everything. Can I fix this PR for you by recompiling the assets ?
> 
> And if you feel like diving into webpack configuration to fix this behavior, you're welcome ! 😄

Yes you can recompile the assets to fix this PR.

Sorry I’m not good in webpack configuration 😔

**Comment by matks** [comment]
> Sorry I’m not good in webpack configuration 😔

I know nobody who is good at it 😆

**Comment by matks** [comment]
@dineshbadrukhiya I updated your PR but I noticed another issue with your PR 😅: you created it on top of a branch you did not update since October:

Capture d’écran 2019-03-13 à 18 23 51

There was a lot of changes in the project since then. Can you recreate this PR from a fresh version of `develop` ? Your `develop` in your fork is too outdated

**Comment by dineshbadrukhiya** [comment]
> @dineshbadrukhiya I updated your PR but I noticed another issue with your PR : you created it on top of a branch you did not update since October:
> 
> Capture d’écran 2019-03-13 à 18 23 51
> 
> There was a lot of changes in the project since then. Can you recreate this PR from a fresh version of `develop` ? Your `develop` in your fork is too outdated

How to update a fresh version? :)

**Comment by matks** [comment]
I assume you have a clone of your fork `https://github.com/dineshbadrukhiya/PrestaShop` on your computer.
If you run `git remote -v` you should see
```
origin	https://github.com/dineshbadrukhiya/PrestaShop.git (fetch)
origin	https://github.com/dineshbadrukhiya/PrestaShop.git (push)
```

You need to add this repository, the official repo, as a new remote, then fetch the latest updates from it. You can do it by running
```
git remote add upstream https://github.com/PrestaShop/PrestaShop.git
git fetch upstream
```

Then you switch the current branch to develop - `git checkout develop` and you pull the latest version of `develop` here - `git pull upstream develop`

To sum up
```
git remote add upstream https://github.com/PrestaShop/PrestaShop.git
git fetch upstream
git checkout develop
git pull upstream develop
```
This will update on your computer your develop branch and make it up-to-date with the official develop branch. Git commands might be a little different for you because it depends on your personal git configuration.

More details in [github help article "syncing a fork"](https://help.github.com/en/articles/syncing-a-fork)

**Comment by matks** [comment]
Closed and replaced by https://github.com/PrestaShop/PrestaShop/pull/12931
